app/cli.php

```diff
@@ -5,6 +5,9 @@
 use Utopia\App;
 use Utopia\CLI\CLI;
 use Utopia\CLI\Console;
+use Utopia\Database\Validator\Authorization;
+
+Authorization::disable();
```

Comment:
leftover?

Reply:
This script includes script from `maintenance.php`, which needs to read from a database. Should this be done in maintanance.php? I thought it would be okay to disable it for all CLI stuff, as CLI does not need authorization.